integration-tests/Makefile

```diff
@@ -0,0 +1,2 @@
+e2e:
```

Comment:
what is the use of this file?

Reply:
just following the Makefile approach / the job will run` make e2e` command to run e2e

Reply:
This is for running locally and nothing to do with the CD pipeline, is that correct?

Reply:
app-sre method follows the make file approach so i did

Reply:
nop

> This is for running locally and nothing to do with the CD pipeline, is that correct?

no the job will trigger this command to run e2e on the pod ref: https://github.com/sawood14012/fabric8-analytics-common/blob/continious-testing/integration-tests/openshift/f8a-e2e-tests.yaml#L38

Reply:
You already specified an entrypoint in your docker image. Why do you need this then?

Reply:
let me remove it

Reply:
done @prashbnair